### PR TITLE
docs: add OpenAI swagger file

### PR DESCRIPTION
**Pull request by hieu-jan**
Add OpenAIAPI.yaml from https://github.com/openai/openai-openapi

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -105,8 +105,12 @@ const config = {
       {
         specs: [
           {
-            spec: "openapi/OpenAPISpec.json", // can be local file, url, or parsed json object
-            route: "/api/",
+            spec: "openapi/OpenAPISpec.json",
+            route: "/api"
+          },
+          {
+            spec: "openapi/OpenAIAPI.yaml", // can be local file, url, or parsed json object
```
Comment (dan-jan) [inline docs/docusaurus.config.js:112]: We should rename this to Jan.yaml, and customize it to fit our API. 

Quick check: was OpenAI's OpenAPI spec under any license? If it is licensed, we may have to re-draft it from scratch. :/

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -105,8 +105,12 @@ const config = {
       {
         specs: [
           {
-            spec: "openapi/OpenAPISpec.json", // can be local file, url, or parsed json object
-            route: "/api/",
+            spec: "openapi/OpenAPISpec.json",
+            route: "/api"
```
Comment (dan-jan) [inline docs/docusaurus.config.js:109]: I would recommend `/api-reference`, as we may use `/api` in the future for an actual API.

File: docs/openapi/OpenAPISpec.json [hunk 3]
```diff
@@ -1,74 +1,74 @@
...
+    "info": {
+      "description": "Jan.ai api reference documentation.",
+      "title": "Rest Endpoints",
+      "version": ""
+    },
+    "paths": {
+      "/api/rest/myquery": {
+        "get": {
+          "summary": "MyQuery",
+          "description": "***\nThe GraphQl query for this endpoint is:\n``` graphql\nquery MyQuery {\n  collections {\n    id\n    name\n    slug\n  }\n}\n```",
```
Comment (dan-jan) [inline docs/openapi/OpenAPISpec.json:12]: What is this GraphQL query for? (seems very outdated)

**Review by dan-jan — CHANGES_REQUESTED** [review]

File: docs/openapi/OpenAPISpec.json (hunk 3 above)
Comment (hieu-jan) [reply docs/openapi/OpenAPISpec.json:12]: This one is an old file from our code base. Just an example of import .json. Maybe the commit changed due to making a new line in the first line. I will remove this file too.

File: docs/docusaurus.config.js (hunk 1 above)
Comment (hieu-jan) [reply docs/docusaurus.config.js:112]: Yeah, it's under an MIT license. Re-draft it from initial is also fit with our need too.

Comment (hieu-jan) [reply docs/docusaurus.config.js:112]: This file is under MIT license. Re-draft it from scratch may be a better option for us as we will customize some parts too

**Comment by dan-jan** [comment]
@hieu-jan Please remove the `yarn.lock` from your PR

**Comment by hieu-jan** [comment]
Re-solved all conversation and conflicts with main branch

**Review by dan-jan — APPROVED** [review]
Preliminary approve

**Pull request merged**